src/components/Filter/ActiveFilterItem.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+import {capitalize} from '../../util';
+import styles from './ActiveFilterItem.scss';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import Button from 'react-bootstrap/lib/Button';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+
+const ActiveFilterItem = React.createClass({
```

Comment:
Siden vi bare har en render-funksjon her, så foreslår jeg at vi gjør dette om til en "pure function" component:
```
const ActiveFilterItem = (props) => {
  (innholdet av render()-funksjonen, bruk props.tagItem og props.onClick istedet for this.props.tagItem osv)
};
```

Forøvrig går det an å gjøre den enda mer lesbar om man ønsker det, ved å skrive
```
const ActiveFilterItem = ({tagItem, onClick}) => {
  (innholdet av render()-funksjonen, bruk tagItem og onClick direkte)
};
```